### ISSUE TITLE
Ingest Content for Content Provider - LANDOE

### ISSUE THREAD

**Issue by lobajuluwa** [gbhl/bhl-europe#324]
Task description: Align (LANDOE) upload data/structure with ingest tool needs

subtask: Ingest (LANDOE) data

```
/mnt/nfs-demeter/upload/providers/at-landoe
```

---
- folder names - number consecutively
- folder structure - seems ok, only one level
- file names:
  - InternalIdentifier - consecutive number from parent folder could may be an internal ID (primary key from database) 
  - FileSequenceNumber - OK
  - PageType - **MISSING**
  - PrintedPageNumber - **MISSING**
- metadata - **MISSING**

**Comment by chris-sleep** [comment]
Current action assessment: OLEF metadata upload, New Directory structure and names

**Comment by akohlbecker** [comment]
Checked folder again for metadata:

```
find . -type f \( ! -iname "*.tif" ! -iname "*.tif.*" ! -iname "*.pdf" \)
./118/.aip/118.guid.txt
./118/.aip/118_document_ocr.pdf.jhove.xml
./118/.aip/118_document_ocr.pdf.txt
./118/.aip/118.scan.log.txt
./118/.aip/118.nfo.rdf.xml
./118/.aip/118.mets.xml
```

so there is no metadata at all

**Comment by chris-sleep** [comment]
Checked against https://bhl.wikispaces.com/Content+_Management and Landoe noted to provide OLEF over OAI-PMH

awaiting OAI URL from CP

**Comment by chris-sleep** [comment]
testing PI tool current handling with uploaded content and identified metadata service: reading files to database to activate content in the tool ongoing

**Comment by chris-sleep** [comment]
retesting with current PI directly on bhl-test1: ongoing issues with timeout/memory limit so initial activation/read step does not finish.

**Comment by wkollernhm** [comment]
@chris-sleep I've updated the PI tool on test. Please check if it now works (as it contains the latest bugfix for paging)

**Comment by chris-sleep** [comment]
I've kicked off the read/analyse step (currrently connecting directly to bhl-test1; provided that behaves I'll test via proxy next
- update after a few seconds; a directory list is now presented quickly and browsing looks rapid.  Initial feeling is good

testing at-landoe/28148 (need to get the SMT/mapping configuration correct, olef generation not yet behaving)

**Comment by chris-sleep** [comment]
@wkollernhm the updated PI paging fix looks good, very quick response now on the analyze step

**Comment by melitabirthaelmer** [comment]
CP finished content upload.
Additional content: landooe_bhl_20120329

**Comment by zhengl** [comment]
@wkollernhm 

All OLEF from LANDOOE is invalid.

The same error is:
error on line 2 at column 74: Namespace prefix xsi for schemaLocation on olef is not defined

Compared with other providers, I don't know why more schema such as xsi:schemaLocation is added into OLEF.

**Comment by wkollernhm** [comment]
The metadata provided by LANDOOE was fixed. The xsi:schemaLocation is a valid information for any schema, since it just defines the download location for the XSD.

**Comment by zhengl** [comment]
But the definition of xsi is missing (xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance")

e.g. http://bhl-int.nhm.ac.uk/static/a0kkwh97/a0kkwh97_olef.xml

**Comment by wkollernhm** [comment]
Yes I know, its been added in the latest updated to the OAI-PMH provider.
